cmd/cephfs/main.go

```diff
@@ -43,6 +43,13 @@ func init() {
 
 func main() {
 
+	err := util.ValidateDriverName(*driverName)
```

Comment:
Validation can cause breakage for implementations that have been using a name other than the default in the driverName parameter, that is not conformant to the current scheme.

Should we call this out in the docs/commit or somewhere? @gman0 @rootfs 

Example: assume, some usage passed in the option "--drivername=my-ceph-rbd", on updating to the latest image, these drivers would fail to start as the name would fail validation

(applies to rbd/main.go as well)

Reply:
with current change AFAIK nothing breaks if we add a validation to checks  `ceph.csi.com`  then the things will start breaking

tested sample  code for validation
```package main

import (
	"fmt"
	"strings"

	"github.com/pkg/errors"
	"k8s.io/apimachinery/pkg/util/validation"
)

func ValidateDriverName(driverName string) error {
	if len(driverName) == 0 {
		return errors.New("driver name is empty")
	}

	if len(driverName) > 63 {
		return errors.New("driver name length should be less than 63 chars")
	}
	var err error
	for _, msg := range validation.IsDNS1123Subdomain(strings.ToLower(driverName)) {
		if err == nil {
			err = errors.New(msg)
			continue
		}
		err = errors.Wrap(err, msg)
	}
	return err
}

func main() {
	driverNames := []string{"csi-rbdplugin", "csi-cephfsplugin", "cephfsplugin.csi.ceph.com", "rbdplugin.csi.ceph.com", "TEST-DRIVER", "TEST_DRIVER123"}
	for _, name := range driverNames {
		err := ValidateDriverName(name)
		if err != nil {
			fmt.Printf("driver name %s failed with err %s\n", name, err)
		}

	}
}
-------------------------------------------------------------
go run main.go 
driver name TEST_DRIVER123 failed with err a DNS-1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*')

```